### ISSUE TITLE
Move the requirement of placing the node into "Maintenance" into the steps section as step 1

### ISSUE THREAD

**Issue by siegfried-h** [NetAppDocs/storagegrid-116#6]
Page: [Upgrade drive firmware using SANtricity System Manager](https://docs.netapp.com/us-en/storagegrid-116/sg6000/upgrading-drive-firmware-using-santricity-system-manager.html)

Please move the requirement of placing the node into "Maintenance" into the steps section as step 1.

So instead of listing this very important information as "What you’ll need" put it as step 1 in the steps section. Many customers are not reading this and start with the drive firmware update immediately. This is known to casue system down and/or data corruption.

Here the section that needs to be moved:

"You have [placed the StorageGRID appliance in maintenance mode](https://docs.netapp.com/us-en/storagegrid-116/sg6000/placing-appliance-into-maintenance-mode.html)."

regards
Siegfried Hepp (Netapp)

**Comment by netapp-madkat** [comment]
Hi Siegfried, 

Thank you for your suggestion. We've made the requested change and should be able to refresh the 11.6 doc site within the next few days.

Kathy

P.S. For consistency with other topics, we decided to leave maintenance mode as a prerequisite and to add a "confirm the appliance is in maintenance mode" step. Now that the maintenance mode requirement is mentioned in both places, it will be hard to miss.
